api/client/clientRouter.js

```diff
@@ -45,10 +51,12 @@ router.delete(':id', (req, res, next) => {
         });
       }
     })
-    .catch(next);
+    .catch(() => {
+      res.status(500).json({ message: 'internal server error' });
+    });
 });
 
-router.put(':id', (req, res, next) => {
+router.put(':id', (req, res) => {
```

Comment:
you added a "/" before :id on the delete request on line 41, why not here as well?